cmake/BuildParameters.cmake

```diff
@@ -124,14 +124,13 @@ if(${PCSX2_TARGET_ARCHITECTURES} MATCHES "i386")
     #     - Only plugins. No package will link to them.
     set(CMAKE_POSITION_INDEPENDENT_CODE OFF)
 
-    if (DISABLE_ADVANCE_SIMD)
-        set(ARCH_FLAG "-msse -msse2 -march=i686")
-    else()
+    if (NOT DISABLE_ADVANCE_SIMD)
```

Comment:
Just rechecked - compiles fine for me on amd64 host using build.sh:
```
./build.sh --release --cross-multilib --no-simd --clean
Doing a clean build.
Building pcsx2 with -DCMAKE_BUILD_PO=FALSE -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=cmake/linux-compiler-i386-multilib.cmake -DDISABLE_ADVANCE_SIMD=TRUE

```
As i see "cmake/linux-compiler-i386-multilib.cmake" do all needed stuff and -march is optimization flag